test/fixtures/ddl/defaultDomains.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = () => ({
+  name: 'generateDDL with default domains',
+  schemas: {},
+  expectedSql: '',
```

Comment:
Could you add a comment that the domains SQL will be added during testing/in the runner to avoid confusion?

Reply:
Should I do it everywhere, or only in file `defaultDomains.js`?